torch/library.py

```diff
@@ -0,0 +1,74 @@
+import torch
+from ._ops import OpOverload
+from typing import Set
+import warnings
+
+__all__ = ['extend_library']
+
+# User created fragment libraries to extend existing libraries
+fragments_for_existing_libraries = {}
```

Comment:
I probably don't need this dictionary

Reply:
nvm I do need this dictionary to ensure that the library object survives outside the scope of a function

Reply:
Why do you want the library to survive outside of a function? It seems to me that the library should get assigned to a global in a module and live that way.